static/src/javascripts/projects/common/modules/commercial/contributions-service.js

```diff
@@ -303,14 +305,16 @@ export const fetchPuzzlesData = async () => {
     const page = config.get('page');
     const payload = await buildBannerPayload();
     const forcePuzzlesBannerTest = isInVariantSynchronous(puzzlesBanner, 'variant');
+    const isPuzzlesBannerSwitchOn = config.get('switches.puzzlesBanner');
```

Comment:
`config.get` can take a second argument which is the fallback value it'll return in the property you want to access isn't available, eg. In the event the switch is accidentally deleted. I think we'd want this to be `false`.

So it'd be updated to `config.get('switches.puzzlesBanner', false)`.